laborator/content/structuri-vectori-siruri/1-2-print-structure/Makefile

```diff
@@ -1,16 +1,2 @@
-NASM = nasm
-AFILES = print_structure.asm 
-OBJS = $(AFILES:.asm=.o) 
-ASM_FLAGS = -f elf32 -g
-LD=gcc
-LDFLAGS = -m32 -g
-BINARIES = print_structure
-
-all : $(BINARIES)
-
-%.o : %.asm
-	$(NASM) $(ASM_FLAGS) -o $@ $<
-
-clean: 
-	rm -f *.o $(BINARIES) *.s
-	rm -f *~ 
\ No newline at end of file
+PROGNAME := print_structure
```

Comment:
Use the same naming convention as for the other exercises / the directory, using `-` instead of `_`.

Reply:
Done